client/src/main/java/jp/co/soramitsu/iroha/java/QueryAPI.java

```diff
@@ -39,106 +41,134 @@ public QueryAPI(IrohaAPI api, Account account) {
 
   private AtomicInteger counter = new AtomicInteger(1);
 
+  private void checkErrorResponse(QueryResponse response) throws ErrorResponseException {
+    if (response.hasErrorResponse()) {
+      ErrorResponse errorResponse = response.getErrorResponse();
+      throw new ErrorResponseException(errorResponse.getErrorCode(),
+          errorResponse.getReason().toString(), errorResponse.getMessage());
```

Comment:
pass errorResponse to constructor and let users who catch this exception to get it (add field ErrorResponse to exception, and add @Getter)